views/movies/new-movie.hbs

```diff
@@ -0,0 +1,21 @@
+<h2>Create movie</h2>
+<form action="/movie/create" method="post">
+  <label for='title'>Name</label>
+  <input type='text' name='name' id='name' />
```

Comment:
Here the key that you have in the movie schema is `title` and not `name` so instead of `name='name'` you should do `name='title'`